torchvision/transforms/v2/functional/_color.py

```diff
@@ -154,6 +139,7 @@ def adjust_saturation_image_tensor(image: torch.Tensor, saturation_factor: float
 
 
 adjust_saturation_image_pil = _FP.adjust_saturation
+_register_kernel_internal(adjust_saturation, PIL.Image.Image)(adjust_saturation_image_pil)
```

Comment:
We have quite a few instances of this pattern. We can leave it at that, since I killed the dispatch test that was previously the reason for me to turn this into a proper function rather than just an alias. OTOH, we want to move the functionality here eventually, so we could also do it now. No strong opinion.

Reply:
```suggestion
@_register_kernel_intern(adjust_saturation, PIL.Image.Image)
def adjust_saturation_image_pil(image, saturation_factor):
     return _FP.adjust_saturation(image, saturation_factor)
```

Reply:
```suggestion
adjust_saturation_image_pil = _register_kernel_internal(adjust_saturation, PIL.Image.Image)(_FP.adjust_saturation)
```

(and kill the line above)